views/checklist.html

```diff
@@ -0,0 +1,162 @@
+<include href="views/navbar.html">
```

Comment:
I would suggest adding some inline comments to the HTML to know what sections the code is pertaining to. Code looks good though!